foodsaving/users/models.py

```diff
@@ -149,9 +141,9 @@ def _send_mail_change_notification(self):
     def _send_welcome_mail(self):
         self._unverify_mail()
 
-        url = '{hostname}/#/verify-mail?key={key}'.format(
+        url = '{hostname}/#/verify-mail?code={code}'.format(
```

Comment:
Ah, the rename from `key` to `code` will break the frontend. Would you rather change the frontend or change it back to `key={code}`?

Also, we could also use the `furl` library to construct the URL (similar to the invitation mail). But not necessary...

Reply:
Oh... sorry for that!
I actually wanted to test everything with my local frontend, then I would have noted that myself. But then I forgot...